conf/audit.yaml

```diff
@@ -76,7 +76,8 @@ log_timing_strategy: pre_logging
 # optional parameters are accepted:
 # - log_format   - Format of the audit record sent to SLF4J. Fields can be configured with bash-style parameter
 #                  substitution. Supported parameters are ${CLIENT_IP}, ${CLIENT_PORT}, ${COORDINATOR_IP}, ${USER},
-#                  ${BATCH_ID}, ${STATUS}, ${OPERATION}, ${OPERATION_NAKED}, ${TIMESTAMP}, and ${SUBJECT}.
+#                  ${BATCH_ID}, ${STATUS}, ${OPERATION}, ${OPERATION_NAKED}, ${SINGLE_LINE_OPERATION}, ${TIMESTAMP}
```

Comment:
Would it be more consistent to use ${OPERATION_SINGLE_LINE}.

Or perhaps ${OPERATION_SQUASHED} or ${OPERATION_SQUEEZE}?